tests/unit/test_masks.py

```diff
@@ -148,11 +159,38 @@ def it_knows_its_mask_tile(self, request):
         tissue_mask_tile_.return_value = expected_mask
 
```

Comment:
not your responsibility, but since you touched this method... this line should also be removed